packages/builders/src/util/normalizeArray.ts

```diff
@@ -5,7 +5,7 @@
  * @param arr - The (possibly variadic) data to normalize
  */
 export function normalizeArray<ItemType>(arr: RestOrArray<ItemType>): ItemType[] {
-	if (Array.isArray(arr[0])) return arr[0];
+	if (Array.isArray(arr[0])) return [...arr[0]];
```

Comment:
just wondering if it is worth updating the documentation to specify that it returns a cloned array